docs/ref/forms/models.txt

```diff
@@ -1,15 +1,246 @@
-====================
-Model Form Functions
-====================
-
-Model Form API reference. For introductory material about model forms, see the
-:doc:`/topics/forms/modelforms` topic guide.
+===========
+Model Forms
+===========
 
 .. module:: django.forms.models
-   :synopsis: Django's functions for building model forms and formsets.
+    :synopsis: ModelForm API reference for inner ``Meta`` class and factory
+        functions
+
+.. class:: ModelForm
+
+.. _ref-modelform:
+
+.. currentmodule:: django.forms
+
+:class:`~django.forms.ModelForm` API reference. For introductory material about
+ModelForms, see the :doc:`/topics/forms/modelforms` topic guide.
+
+.. _ref-modelform-meta-options:
+
+ModelForm ``Meta`` API
+======================
+
+.. class:: ModelFormOptions
+
+The ``_meta`` API is used to build forms that reflect a Django model. The API
+is accessible through the ``_meta`` attribute of each model form, which is an
+instance of a ``django.forms.models.ModelFormOptions`` object.
+The exact structure of the form produced can be controlled by setting metadata
+options as attributes of an inner ``Meta`` class.
+
+Here is an example using some sort of Book model::
+
+    from django.forms import ModelForm
+    from myapp.models import Book
+
+
+    class BookForm(ModelForm):
+        class Meta:
+            model = Book
+            fields = ["title", "author"]
+            help_texts = {
+                "title": "The title of the book",
+                "author": "The author of the book",
+            }
+            # ... other attributes
+
+Required attributes are :attr:`~ModelFormOptions.model`, and either
+:attr:`~ModelFormOptions.fields` or :attr:`~ModelFormOptions.exclude`. All
+other ``Meta`` attributes are optional.
+
+Optional attributes, other than :attr:`~ModelFormOptions.localized_fields` and
+:attr:`~ModelFormOptions.formfield_callback`, expect a dictionary that maps a
+model field name to a value. Any field that is not defined in the dictionary
+falls back to the field's default value.
+
+.. admonition:: Invalid field names
+
+    An invalid or excluded field name strings in an optional dictionary
+    attribute will have no effect since fields that are not included are not
+    accessed.
```

Comment:
```suggestion
    Invalid or excluded field names in an optional dictionary attribute have no
    effect, since fields that are not included are not accessed.
```
(perhaps)